src/ontology/fbbt-edit.obo

```diff
@@ -162080,12 +162090,13 @@ creation_date: 2017-04-28T16:08:36Z
 
 [Term]
 id: FBbt:00111762
-name: lobula complex columnar neuron LPLC1
+name: lobula plate-lobula columnar neuron LPLC1
 def: "An extrinsic columnar neuron whose cell body lies in the dorsal area of the lateral cell body rind. It has its main dendritic arbors in lobula layers 2 to 4, and 5B, with lower density in layer 3. Presynaptic sites are observed in the boundary of lobula layers 3/4. Its processes extend the full length of the lobula plate, being sparser in layer 1. It projects to a large optic glomerulus of the PVLP, posterior to LPLC2 and dorsal to LC4, and located at the PVLP and PLP border. There are around 62 cells of this type. They are cholinergic (Davis et al., 2020)." [FlyBase:FBrf0233032, FlyBase:FBrf0234700, FlyBase:FBrf0244935]
 comment: Pre- versus postsynaptic innervation assessed with a presynaptic marker (synaptotagmin-HA driver) (Wu et al., 2016).
+synonym: "lobula complex columnar neuron LPLC1" EXACT [FBC:DPG]
```

Comment:
I wouldn't put your initials on these - you aren't a really reference for the synonym. It is ok to leave blank (pretty obvious from the name that it means the same thing).